libsolidity/interface/CompilerStack.cpp

```diff
@@ -242,7 +242,7 @@ bool CompilerStack::parse()
 	}
 	if (Error::containsOnlyWarnings(m_errorReporter.errors()))
 	{
-		m_stackState = ParsingSuccessful;
+		m_stackState = ParsingPerformed;
```

Comment:
Why only set it if there are no errors? I thought the idea was to set it to "Performed" if it has been done, even if there were errors?

Reply:
Changed now.